extensions/binding/org.eclipse.smarthome.binding.hue.test/src/test/groovy/org/eclipse/smarthome/binding/hue/test/HueBridgeHandlerOSGiTest.groovy

```diff
@@ -228,7 +228,7 @@ class HueBridgeHandlerOSGiTest extends OSGiTest {
     }
 	
     @Test
-    void 'assert that a status configuration message for missing bridge IP is properly returned (IP is null)'() {
+    void 'assert that a status configuration message for missing bridge IP is properly returned - IP is null'() {
```

Comment:
I had some problems with FindBugs, this method name caused FindBugs to crash.
The Groovy method is compiled in a separate .class and when FindBugs checks the .class file it does not expect the class name to contain braces.